docs/source/topics/calculations/usage.rst

```diff
@@ -495,17 +495,17 @@ The contract of the 'retrieve temporary list' is essentially that the files will
 
 .. _topics:calculations:usage:calcjobs:stashing:
 
-Stashing files on the remote
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stashing on the remote
+~~~~~~~~~~~~~~~~~~~~~~
 
 .. versionadded:: 1.6.0
 
-The ``stash`` option namespace allows a user to specify certain files that are created by the calculation job to be stashed somewhere on the remote.
-This can be useful if those files need to be stored for a longer time than the scratch space where the job was run is typically not cleaned for, but need to be kept on the remote machine and not retrieved.
+The ``stash`` option namespace allows a user to specify certain files and/or folders that are created by the calculation job to be stashed somewhere on the remote where the job is run.
+This can be useful if these need to be stored for a longer time on a machine where the scratch space is cleaned regularly, but they need to be kept on the remote machine and not retrieved.
 Examples are files that are necessary to restart a calculation but are too big to be retrieved and stored permanently in the local file repository.
 
-The files that are to be stashed are specified through their relative filepaths within the working directory in the ``stash.source_list`` option.
-Using the ``COPY`` mode, the target path defines another location (on the same filesystem as the calculation) to copy these files to, and is set through the ``stash.target_base`` option, for example:
+The files and that need to be stashed are specified through their relative filepaths within the working directory in the ``stash.source_list`` option.
```

Comment:
```suggestion
The files/folder that need to be stashed are specified through their relative filepaths within the working directory in the ``stash.source_list`` option.
```

I would maybe just use it here too because it is specifically saying how to specify them in the `stash.source_list`